intake/interface/source/defined_plots.py

```diff
@@ -94,7 +123,9 @@ def source(self, source):
         BaseView.source.fset(self, source)
         if self.select:
             self.select.options = self.options
-        if source and dfviz and source.container == 'dataframe':
+        if source and hvplot and source.container == 'dataframe':
+            self.custom.disabled = False
+        elif source and dfviz and source.container == 'dataframe':
```

Comment:
dfviz is not well maintained and should probably be retired. This is OK for now, but following the ext release, we will probably retire dfviz so that it instructs users to get latest hvplot instead.

Reply:
If we make hvplot required, then removing dfviz should be okay.